packages/core/src/new-api/internal/constants.ts

```diff
@@ -0,0 +1,28 @@
+import { DeployConfig } from "../types/deployer";
+
+/**
+ * The interval, in milliseconds, between checks to see if a new block
+ * has been created
+ */
+const DEFAULT_BLOCK_POLLING_INTERVAL = 200;
```

Comment:
I think we should do something like `1000` initially, as most chains will take seconds to mine a block.